src/modules/github.ts

```diff
@@ -1,29 +1,24 @@
-// -----------------------------------------------------------------------------------------
-// #region Imports
-// -----------------------------------------------------------------------------------------
-
-const { createNetrcAuth } = require("octokit-auth-netrc");
-const { Octokit } = require("@octokit/rest");
-const { StringUtils } = require("andculturecode-javascript-core");
-const constants = require("./constants");
-const echo = require("./echo");
-const formatters = require("./formatters");
-const fs = require("fs");
-const git = require("./git");
-const js = require("./js");
-const os = require("os");
-const shell = require("shelljs");
-const upath = require("upath");
-const userPrompt = require("./user-prompt");
-
-// #endregion Imports
+import { StringUtils } from "andculturecode-javascript-core";
```

Comment:
So upon further testing of the issue with exits hanging, it appears to be effecting both the `github` and `workspace` command series. The common denominator of course is the `github` module. There is likely something in here causing the exits to hang. It does eventually exit. Just takes several minutes.

Reply:
Thanks for digging into that 👍 I'll take a further look